styles/signals.mapcss

```diff
@@ -1349,8 +1347,8 @@ node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:combined"=
 /*  - can display Hl 11, Hl 12a               */
 /*  - don't have to be able to display Hl 12b */
 /**********************************************/
-node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:combined"="DE-ESO:hl"]["railway:signal:combined:form"=light]["railway:signal:combined:states"]["railway:signal:combined:states"=~/hl12a/]["railway:signal:combined:states"=~/hl11/]["railway:signal:combined:states"!~/hl12b/],
-node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:combined"="DE-ESO:hl"]["railway:signal:combined:form"=light]["railway:signal:combined:states"]["railway:signal:combined:states"=~/hl12a/]["railway:signal:combined:states"=~/hl11/]["railway:signal:combined:states"=~/hl12b/]
+node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:combined"="DE-ESO:hl"]["railway:signal:combined:form"=light]["railway:signal:combined:states"]["railway:signal:combined:states"=~/^(.*;)?DE-ESO:hl12a(;.*)?$/]["railway:signal:combined:states"=~/^(.*;)?DE-ESO:hl11(;.*)?$/]["railway:signal:combined:states"!~/^(.*;)?DE-ESO:hl12b(;.*)?$/]
```

Comment:
In fact, the following line is just wrong, as it is the opposite as this one regarding Hl12b. It's the same error as we already had for Hl3b.